c3d/c3d.py

```diff
@@ -1115,7 +1115,18 @@ def add_group(self, group_id, name, desc):
             raise KeyError(group_id)
         name = name.upper()
         if name in self._groups:
-            raise KeyError(name)
+            #In some cases group name is not unique (though c3d spec requires that).
+            #To allow using such files we auto-generate new name. 
+            #Notice that refering to this group's parameters later with the original name will fail.
```

Comment:
Comments should start "# "
```suggestion
            # In some cases group name is not unique (though c3d spec requires that).
            # To allow using such files we auto-generate new name. 
            # Notice that referring to this group's parameters later with the original name will fail.
```